profiling/src/profiling/mod.rs

```diff
@@ -488,9 +488,20 @@ pub enum UploadMessage {
 
 impl Profiler {
     pub fn new(output_pprof: Option<Cow<'static, str>>) -> Self {
+        cfg_if::cfg_if! {
+            if #[cfg(feature = "trigger_time_sample")] {
+                // We don't want giant profiles when benchmarking.
+                // We don't want it to block, either.
+                // 1 fits both goals.
+                let message_capacity = 1;
+            } else {
+                let message_capacity = 128;
```

Comment:
We might need to decouple the `message_capacity` from the `trigger_time_sample` feature. The prof-correctness tests are build with `trigger_time_sample` but some of the tests are now failing, as the capacity of the channel is too small and messages are being dropped.

Reply:
Doh, that's obvious in retrospect 🤦🏻